store/videos.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable prettier/prettier */
+export const state = () => {
```

Comment:
just small, you could also use the shorter notation: export const state = () => ({})